components/camera/ultrasonic/ultrasonic_test.go

```diff
@@ -0,0 +1,55 @@
+package ultrasonic
```

Comment:
create a new test where you created an inject.Sensor where you define a simple "reading" method and use that to test out NextPointCloud and Stream

Reply:
for example:
```
fakeUS := inject.NewSensor("mySensor")
fakeUS.ReadingsFunc = func(ctx context.Context, extra map[string]interface{}) (map[string]interface{}, error) {
   return map[string]interface{}{"distance": 3.2}, nil
}
```